api/opencl_runtime_layer.asciidoc

```diff
@@ -6374,6 +6408,19 @@ include::{generated}/api/version-notes/CL_MEM_D3D11_RESOURCE_KHR.asciidoc[]
       returns the _resource_ argument specified when _memobj_ was created.
 endif::cl_khr_d3d11_sharing[]
 
+ifdef::cl_ext_buffer_device_address[]
+| {CL_MEM_DEVICE_ADDRESS_EXT_anchor}
+
+include::{generated}/api/version-notes/CL_MEM_DEVICE_ADDRESS_EXT.asciidoc[]
+  |  {cl_mem_device_address_EXT_TYPE}[]
+    | If _memobj_ was created using {clCreateBufferWithProperties} with
+      the {CL_MEM_DEVICE_PRIVATE_ADDRESS_EXT} property set to CL_TRUE,
+      returns a list of device addresses for the buffer, one for each
+      device in the context.
```

Comment:
Do we need to say anything about the order in which the device addresses are returned?  For example, are the device addresses returned in the same order as the list of devices passed to `clCreateContext` and/or returned by `CL_CONTEXT_DEVICES`?